public/toolkit/styles/molecules/_tabs.scss

```diff
@@ -19,43 +18,34 @@
       color: $jet;
       box-shadow: inset 0 -2px 0 0 $primary;
     }
+    @include a-center;
+    text-align: center;
+    width: 100%;
   }
 
   &.active > a,
   &.active > button {
     box-shadow: inset 0 2px 0 0 $primary;
     color: $tabs-navigation-active-font-color;
+    background: white;
     @include type-weight(semi);
     &:hover {
       box-shadow: inset 0 2px 0 0 $primary;
     }
   }
 }
 
-// Small full width
-.full-width-small-only {
-  @media #{$small-only} {
-    &.tabs dd, 
-    &.tabs .tab-title {
-      width: 100%;
-
-      > a,
-      > button {
-        text-align: center;
-
-        &:hover {
-          box-shadow: none;
-        }
-      }
-
-      &.active > a,
-      &.active > button {
-        box-shadow: none;
+// Switch tabs to full width buttons for medium and small screens
+.stretch-to-fill-medium-down {
```

Comment:
we're really only using these styles directly in the pattern lib components in partners. Questioning the utility of having this pulled out into a separate class

Reply:
I kind of have an issue with the class name here, it seems pretty generic and makes it sounds like it'll fill to 100% at medium down, when really it fills to 50% at medium down.

maybe this selector could be `.tabs.stretch-to-fill-medium-down`?

Reply:
When you say `Questioning the utility of having this pulled out into a separate class`, what's the alternative you're thinking of? Having these styles live in partners?

Reply:
Wondering if I should just pull it into the tab-title class above since we use this on the one component in partners that uses the tabs css classes

Reply:
Oh yeah that's fine and makes sense to me!